aries-site/src/examples/templates/list-views/ListScreenExample.js

```diff
@@ -179,14 +171,15 @@ const ScreenContainer = ({ mobile, ...rest }) => {
 export const ListScreenExample = ({ ...rest }) => (
   <ScreenContainer {...rest}>
     <AppHeaderExample />
-    <PageHeaderExample title="User Controls" />
+    <PageHeader
+      title="User Controls"
+      actions={<Button label="Manage" primary />}
+      margin={{ vertical: 'medium' }}
```

Comment:
```suggestion
      pad={{ vertical: 'medium' }}
```
Any reason not to do pad?

Reply:
nope. Missed it.

Reply:
Done.